dokka-subprojects/plugin-base/src/main/kotlin/org/jetbrains/dokka/base/transformers/documentables/JvmMappedMethodsDocumentableFilterTransformer.kt

```diff
@@ -256,15 +257,21 @@ internal class JvmMappedMethodsDocumentableFilterTransformer(context: DokkaConte
     } ?: false
 
     // user case: stdlib
-    private fun DFunction.isOnlyInJVM() = sourceSets.all { it.analysisPlatform == Platform.jvm }
+    private fun Documentable.isOnlyInJVM() = sourceSets.all { it.analysisPlatform == Platform.jvm }
+
+    private fun DFunction.shouldBeSuppressed() = if (this.hasDeprecated())
+        true
+    else {
+        val fqn = this.dri.let { it.packageName + "." + it.classNames + "." + it.callable?.name }
+        fqn in VISIBLE_METHOD_SIGNATURES || fqn in MUTABLE_METHOD_SIGNATURES || fqn in DEPRECATED_LIST_METHODS || fqn in HIDDEN_METHOD_SIGNATURES || fqn in NOT_CONSIDER_METHOD_SIGNATURES
+    }
 
     override fun shouldBeSuppressed(d: Documentable): Boolean =
-        if (d is DFunction && d.isOnlyInJVM()) {
-            if (d.hasDeprecated())
-                true
-            else {
-                val fqn = d.dri.let { it.packageName + "." + it.classNames + "." + it.callable?.name }
-                fqn in VISIBLE_METHOD_SIGNATURES || fqn in MUTABLE_METHOD_SIGNATURES || fqn in DEPRECATED_LIST_METHODS || fqn in HIDDEN_METHOD_SIGNATURES || fqn in NOT_CONSIDER_METHOD_SIGNATURES
+        if (d.isOnlyInJVM()) {
+            when (d) {
+                is DFunction -> d.shouldBeSuppressed()
+                is DProperty -> d.getter?.shouldBeSuppressed() ?: false
```

Comment:
What about `setter`?
Probably, we don't need to check for it, but it would be nice to have a comment here on why